src/MgvLib.sol

```diff
@@ -138,6 +139,8 @@ interface HasMgvEvents {
 
   event OfferFailWithPosthookData(
     bytes32 indexed olKeyHash,
+    address indexed taker,
```

Comment:
If taker/maker is deemed too much data, we can consider removing maker and having id be indexed - maker would have to make a query for each of their offers, but they could get the data.
Maybe you can use Espen's newly added gas diff feature and see what the effect of these changes are :)

Reply:
Though do note that in the past we emitted outbound/inbound indexed, so this is not much more.
One could argue that for something like MangroveOrder indexed offerId could make more sense to query on for the UI.

Reply:
This may be the case for most events emitting maker an id. Maybe @adhusson or @espendk has an opinion.

Reply:
Would it be cheaper to emit `id` instead of `maker`?

For `MangroveOrder`, I don't think it'll make much difference: Yes, the UI will receive events for offer ids that don't belong to the current user, but it'll be easy to filter on and the number of events will likely not be very high.

Reply:
After a talk with Lasse. I then agree that by indexing the id, it will be easier for RPC call filter correctly. As the maker could be a Strat, like MangroveOrder, then it is not really as relevant as the actual id. This way the Strat can emit the relation between the maker and the offerId. It is then doable to do RPC calls and filter on the relevant ids.